lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -259,3 +256,78 @@ def test_url_tick():
     for i in range(len(ax.yaxis.get_major_ticks())):
         assert f'http://example.com/{i}'.encode('ascii') in b1
     assert b1 == b2
+
+
+def test_svg_default_metadata(monkeypatch):
+    # Values have been predefined for 'Creator', 'Date', 'Format', and 'Type'.
+    monkeypatch.setenv('SOURCE_DATE_EPOCH', '19680801')
+
+    fig, ax = plt.subplots()
+    with BytesIO() as fd:
+        fig.savefig(fd, format='svg')
+        buf = fd.getvalue().decode()
+
+    # Creator
+    assert mpl.__version__ in buf
+    # Date
+    assert '1970-08-16' in buf
+    # Format
+    assert 'image/svg+xml' in buf
+    # Type
+    assert 'StillImage' in buf
+
+
+def test_svg_metadata():
+    single_value = ['Coverage', 'Identifier', 'Language', 'Relation', 'Source',
+                    'Title', 'Type']
+    multi_value = ['Contributor', 'Creator', 'Keywords', 'Publisher', 'Rights']
+    metadata = {
+        'Date': [datetime.date(1968, 8, 1),
+                 datetime.datetime(1968, 8, 2, 1, 2, 3)],
+        'Description': 'description\ntext',
+        **{k: f'{k} foo' for k in single_value},
+        **{k: [f'{k} bar', f'{k} baz'] for k in multi_value},
+    }
+
+    fig, ax = plt.subplots()
+    with BytesIO() as fd:
+        fig.savefig(fd, format='svg', metadata=metadata)
+        buf = fd.getvalue().decode()
+
+    SVGNS = '{http://www.w3.org/2000/svg}'
+    RDFNS = '{http://www.w3.org/1999/02/22-rdf-syntax-ns#}'
+    CCNS = '{http://creativecommons.org/ns#}'
+    DCNS = '{http://purl.org/dc/elements/1.1/}'
+
+    root = xml.etree.ElementTree.fromstring(buf)
+    rdf = root.find(f'./{SVGNS}metadata/{RDFNS}RDF')
+
+    # Check things that are single entries.
+    assert root.find(f'./{SVGNS}title').text == metadata['Title']
```

Comment:
perhaps use findall() and check that the result is `[metadata['Title']]` (i.e. also check that there's a single entry while you're at it)?
likewise below

Reply:
Yea, I changed the others, but should do these too.

Reply:
I re-wrote them all to use a similar list-based check.